deploy/chart/templates/daemonset.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ccm-linode
+  labels:
+    app: ccm-linode
+  namespace: {{ required ".Values.namespace required" .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: ccm-linode
+  template:
+    metadata:
+      labels:
+        app: ccm-linode
+    spec:
+      serviceAccountName: ccm-linode
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+      {{- end }}
+      hostNetwork: true
+      containers:
+        - image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
```

```suggestion
        - image: {{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.appVersion }}
```
I think we should tie the default image tag to `.Chart.appVersion`. We can leave `.Values.image.tag` (e.g. `latest`) as an override for the user to specify and not use `latest` as a default since it's [better practice to use explicit version tags instead](https://cloud.ibm.com/docs/Registry?topic=Registry-troubleshoot-docker-latest).